README.md

```diff
@@ -9,16 +9,16 @@ This package enriches your Fivetran data by doing the following:
 - Models staging tables, which will be used in our transform package
 
 ## Models
-This package contains staging models, designed to work simultaneously with our [Apple Search Ads modeling package](https://github.com/fivetran/dbt_apple_search_ads).  The staging models:
+This package contains staging models, designed to work simultaneously with our [Apple Search Ads modeling package](https://github.com/fivetran/dbt_apple_search_ads).  The staging models are designed to:
```

Comment:
@fivetran-joemarkiewicz In line 9 above, we say "transform package," but here we say "modeling package." I assume this refers to the same thing. I think we should pick one term and use it throughout. (I can work on applying this change to other READMEs once we make a decision.)

Reply:
Great call out. I agree we should be consistent. My opinion is to stick with "Modeling". This allows us to also align with the "Data Models" term we are using for product promotions.

Reply:
@fivetran-joemarkiewicz Our [new transformations docs](https://beta.fivetran.com/docs/transformations/dbt/data-models) call these "transform models." Could that work for the READMEs?

Reply:
Yeah that would work perfect!

Reply:
Great, thanks! I've created a follow-up issue for our docs team to make this change throughout the READMEs: https://fivetran.height.app/T-190128